js/ui/lookup.js

```diff
@@ -1036,28 +1031,10 @@ const Lookup = DropDownList.inherit({
         this.callBase();
     },
 
-    _setDeprecatedOptions: function() {
-        this.callBase();
-
-        extend(this._deprecatedOptions, {
-            'title': { since: '20.1', alias: 'dropDownOptions.title' },
-            'titleTemplate': { since: '20.1', alias: 'dropDownOptions.titleTemplate' },
-            'onTitleRendered': { since: '20.1', alias: 'dropDownOptions.onTitleRendered' },
-            'fullScreen': { since: '20.1', alias: 'dropDownOptions.fullScreen' },
-            'popupWidth': { since: '20.1', alias: 'dropDownOptions.width' },
-            'popupHeight': { since: '20.1', alias: 'dropDownOptions.height' },
-            'shading': { since: '20.1', alias: 'dropDownOptions.shading' },
-            'closeOnOutsideClick': { since: '20.1', alias: 'dropDownOptions.closeOnOutsideClick' },
-            'position': { since: '20.1', alias: 'dropDownOptions.position' },
-            'animation': { since: '20.1', alias: 'dropDownOptions.animation' }
-        });
-    },
-
     _optionChanged: function(args) {
-        const name = args.name;
-        const value = args.value;
+        const { name, fullName, value } = args;
 
-        switch(name) {
+        switch(fullName) {
```

Comment:
I don't think it's a good idea: `name` is used in all other components

Reply:
ok